data/simpy.gram

```diff
@@ -20,7 +20,7 @@ small_stmt[stmt_ty]:
     | break_stmt
     | continue_stmt
     | assignment
-    | e=expressions { _Py_Expr(e, EXTRA(e, e)) }
+    | e=expressions { _Py_Expr(e, EXTRA(e, expr_type, e, expr_type)) }
```

Comment:
Why not `EXTRA_EXPR(e, e)`?

Reply:
Just an oversight from my side, I guess.